libsql-server/src/auth/user_auth_strategies/jwt.rs

```diff
@@ -80,6 +88,20 @@ mod jwt_time {
     }
 }
 
+fn validate_any_jwt(
+    jwt_keys: &Vec<jsonwebtoken::DecodingKey>,
+    jwt: &str,
+) -> Result<Authenticated, AuthError> {
+    for (index, jwt_key) in jwt_keys.iter().enumerate() {
```

Comment:
Should this be limited to two keys so that no iteration needs to be done and encapsulate the concept some how. 

EG:
validate_jwt(&primary_jwt_key, jwt) || validate_jwt(&secondary_jwt_key, jwt);

Reply:
It's a good question. Here was my thinking:

1) For the (99%) key rotation use case it should only be in this state for a relatively short period of time (maybe a few hours or a day tops). Otherwise it'll be in the singleton case where we use the `DecodingKeyContainer::Single` to avoid iteration per Turso's suggestion.
2) There may be use cases that I'm not thinking about where 3 or more keys are actually desired. I couldn't come up with a good reason to artificially block this.